test/kata1.fizzBuzz.test.js

```diff
@@ -1,11 +1,34 @@
 const { fizzBuzz } = require("../src");
 
 describe("fizzBuzz", () => {
-  test("returns Fizz when passed a multiple of 3", () => {});
+  test("returns Fizz when passed a multiple of 3", () => {
+    expect(fizzBuzz(3)).toEqual('Fizz');
+    expect(fizzBuzz(6)).toEqual('Fizz');
+    expect(fizzBuzz(9)).toEqual('Fizz');
+    expect(fizzBuzz(12)).toEqual('Fizz');
+  });
+  
+  test("returns Buzz when passed a multiple of 5", () => {
+    expect(fizzBuzz(5)).toEqual('Buzz');
+    expect(fizzBuzz(10)).toEqual('Buzz');
+    expect(fizzBuzz(25)).toEqual('Buzz');
+    expect(fizzBuzz(20)).toEqual('Buzz');
+  });
```

Comment:
You seem to be using 4 spaces of indentation more than anything else here, so you should use the same number here too:
```
    test("returns Fizz when passed a multiple of 3", () => {
        expect(fizzBuzz(3)).toEqual('Fizz');
        expect(fizzBuzz(6)).toEqual('Fizz');
        expect(fizzBuzz(9)).toEqual('Fizz');
        expect(fizzBuzz(12)).toEqual('Fizz');
    });
  
    test("returns Buzz when passed a multiple of 5", () => {
        expect(fizzBuzz(5)).toEqual('Buzz');
        expect(fizzBuzz(10)).toEqual('Buzz');
        expect(fizzBuzz(25)).toEqual('Buzz');
        expect(fizzBuzz(20)).toEqual('Buzz');
    });
```